### ISSUE TITLE
Master doesn't build.

### ISSUE THREAD

**Issue by jasedit** [ECE4574-5574/ServerAPI#12]
Looks like a set of files were deleted, without updating the HomeAutomationServer/HomeAutomationServer.csproj file.

**Comment by jasedit** [comment]
For reference:

```
Build FAILED.
Errors:

/Users/jpz/src/ece5574/ece5574.sln (default targets) ->
(Build target) ->
/Users/jpz/src/ece5574/ServerAPI/HomeAutomationServer/HomeAutomationServer.csproj (default targets) ->
/Library/Frameworks/Mono.framework/Versions/3.12.1/lib/mono/4.5/Microsoft.CSharp.targets (CoreCompile target) ->

    CSC: error CS2001: Source file `Models/House.cs' could not be found
    CSC: error CS2001: Source file `Models/Device.cs' could not be found
    CSC: error CS2001: Source file `Models/Space.cs' could not be found
    CSC: error CS2001: Source file `Models/User.cs' could not be found

     0 Warning(s)
     4 Error(s)
```

If I remove those files from the build, and add `Models/AppCache.cs`, then I get:

```
Errors:

/Users/jpz/src/ece5574/ece5574.sln (default targets) ->
(Build target) ->
/Users/jpz/src/ece5574/ServerAPI/HomeAutomationServer/HomeAutomationServer.csproj (default targets) ->
/Library/Frameworks/Mono.framework/Versions/3.12.1/lib/mono/4.5/Microsoft.CSharp.targets (CoreCompile target) ->

    Controllers/StorageController.cs(102,37): error CS1061: Type `HomeAutomationServer.Services.DeviceRepository' does not contain a definition for `SaveDevice' and no extension method `SaveDevice' of type `HomeAutomationServer.Services.DeviceRepository' could be found. Are you missing an assembly reference?
    Controllers/StorageController.cs(146,36): error CS1061: Type `HomeAutomationServer.Services.SpaceRepository' does not contain a definition for `saveSpace' and no extension method `saveSpace' of type `HomeAutomationServer.Services.SpaceRepository' could be found. Are you missing an assembly reference?

     66 Warning(s)
     2 Error(s)
```

So something is going on.

**Comment by jasedit** [comment]
Fixed as of the merge from #11.

**Comment by suwarna07** [comment]
I am not able to build master.
